test/e2e/runner.go

```diff
@@ -84,7 +84,7 @@ func main() {
 		os.Setenv("KUBECONFIG", cfg.GetKubeConfig())
 		log.Printf("Kubeconfig:%s\n", cfg.GetKubeConfig())
 		log.Println("Waiting on nodes to go into ready state...")
-		ready := node.WaitOnReady(eng.NodeCount(), 10*time.Second, 10*time.Minute)
+		ready := node.WaitOnReady(eng.NodeCount(), 10*time.Second, 15*time.Minute)
```

Comment:
do we want to make this `cfg.Timeout`? that way we can configure it as an env var?